### ISSUE TITLE
Broken download

### ISSUE THREAD

**Issue by chrisgorgo** [OpenNeuroOrg/openneuro#839]
When trying to download: https://openneuro.org/datasets/ds001454/versions/1.0.0

```
zipEntry.js:292 Uncaught TypeError: Cannot read property 'length' of null
    at t (zipEntry.js:292)
    at y.e.zipFiles.o._read (zipEntry.js:292)
    at y.read (zipEntry.js:292)
    at R (zipEntry.js:292)
    at zipEntry.js:292
    at p.run (zipEntry.js:292)
    at d (zipEntry.js:292)
```

This could be limited to this one dataset.

**Comment by chrisgorgo** [comment]
This dataset is also missing on s3://openneuro.org

**Comment by chrisgorgo** [comment]
Scratch the last comment - I can find it on s3://openneuro.org now (had to refresh).

**Comment by chrisgorgo** [comment]
However it seems to be missing on github: https://github.com/OpenNeuroDatasets?utf8=%E2%9C%93&q=ds001454&type=

**Comment by nellh** [comment]
Downloads are fixed for this dataset, I cleared the cache entry for this dataset's latest revision (other revisions looks fine).

The underlying bug is we have some situation which updates the cache to include the S3 files before it is done exporting, leading to files with null download URLs. Will need to investigate that part further.

**Comment by nellh** [comment]
This is fixed by #842
